client/src/utils/apiWrapper.js

```diff
@@ -29,6 +29,60 @@ export const getSampleResponse = () => {
     }));
 };
 
+/**
+ * Returns attending sessions of user
+ * Returns GET_SAMPLE_FAIL upon failure
+ */
+export const getAttendingSessions = (userId) => {
+  const requestString = `${BASE_URL}/session/attending/${userId}`;
+  return axios
+    .get(requestString, {
+      headers: {
+        'Content-Type': 'application/JSON',
+      },
+    })
+    .catch((error) => ({
+      type: 'GET_SAMPLE_FAIL',
+      error,
+    }));
+};
+
+/**
+ * Returns user by id
+ * Returns GET_SAMPLE_FAIL upon failure
+ */
+export const getUserById = (userId) => {
+  const requestString = `${BASE_URL}/user/${userId}`;
+  return axios
+    .get(requestString, {
+      headers: {
+        'Content-Type': 'application/JSON',
+      },
+    })
+    .catch((error) => ({
+      type: 'GET_SAMPLE_FAIL',
+      error,
+    }));
+};
```

Comment:
You can also remove this thanks to memberDB tool! You never need to call this anymore